airflow/www/templates/airflow/dags.html

```diff
@@ -57,7 +57,8 @@
       Airflow found incompatible data in the <code>{{ original_table_name }}</code> table in the
       metadatabase, and has moved them to <code>{{ moved_table_name }}</code> during the database migration
       to upgrade. Please inspect the moved data to decide whether you need to keep them, and manually drop
-      the <code>{{ moved_table_name }}</code> table to dismiss this warning.
+      the <code>{{ moved_table_name }}</code> table to dismiss this warning.<p/>
```

Comment:
```suggestion
      the <code>{{ moved_table_name }}</code> table to dismiss this warning.</p>
```
`p` shouldn't be a self closing tag. We should either wrap both sections in `p`, or use ` br` here. I don't have a strong preference which.